kong/clustering/wrpc_control_plane.lua

```diff
@@ -173,6 +174,11 @@ _M.check_configuration_compatibility = clustering_utils.check_configuration_comp
 
 
 function _M:handle_cp_websocket()
+  if get_method() == "HEAD" then
```

Comment:
@hbagdi any reservations? This will essentially add a new clustering API behavior (`HEAD /v1/wrpc` => HTTP 200), which I assume would need to make it into the [wRPC handler in koko](https://github.com/Kong/koko/blob/f407ae7363989aca9e08b8808dd5efd05ebea7b7/internal/server/kong/ws/server.go#L153).

Before, `HEAD /v1/wrpc` would return HTTP 400 for missing some required params. Short-circuiting on `HEAD` allows us to make the data-plane feature detection more explicit (see the change to `kong/clustering/utils.lua` above).